File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task33_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.*;

public class QueueTime {
    public static int queueTime(int[] customers, int n) {
        int[] tills = new int[n];
        for (int time : customers) {
            int minIndex = 0;
            for (int i = 1; i < n; i++) {
                if (tills[i] < tills[minIndex]) {
                    minIndex = i;
                }
            }
            tills[minIndex] += time;
        }
        int maxTime = 0;
        for (int time : tills) {
            if (time > maxTime) {
                maxTime = time;
            }
        }
        return maxTime;
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task33_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <vector>
#include <algorithm>

int queueTime(std::vector<int> customers, int n) {
    if (customers.empty()) {
        return 0;
    }
    std::vector<int> tills(n, 0);
    for (int time : customers) {
        int minTill = std::distance(tills.begin(), std::min_element(tills.begin(), tills.end()));
        tills[minTill] += time;
    }
    return *std::max_element(tills.begin(), tills.end());
}